### ISSUE TITLE
One of the best tools i've used but one question.

### ISSUE THREAD

**Issue by TechTerror187** [jorhelp/Ingram#65]
how do i change default passwords attack inputs these are shown in log.txt: "passwords=['admin', 'admin12345', 'asdf1234', 'abc12345', '12345admin', '12345abc']". is it possible to change them? or even increase their amount? if not this tool would be perfect if it had this function.

**Comment by jorhelp** [comment]
yes, you can change the config params in the file "Ingram/config.py"

**Comment by TechTerror187** [comment]
Thank you!

**Comment by halstead7** [comment]
Did you solve this problem ? error occurred, see the log.txt for more information???

**Comment by TechTerror187** [comment]
> Did you solve this problem ? error occurred, see the log.txt for more information???

depends on what the log.txt says about it, make sure ur syntax is correct when editing code if not remove package and install it again and every setting will be default

**Comment by InvisibleRagman** [comment]
> > Did you solve this problem ? error occurred, see the log.txt for more information???
> 
> depends on what the log.txt says about it, make sure ur syntax is correct when editing code if not remove package and install it again and every setting will be default

The log file is always empty on this error.

**Comment by TechTerror187** [comment]
> > > Did you solve this problem ? error occurred, see the log.txt for more information???
> > 
> > 
> > depends on what the log.txt says about it, make sure ur syntax is correct when editing code if not remove package and install it again and every setting will be default
> 
> The log file is always empty on this error.

i never had that error even when i first time used the program make sure to install requirements if they are not installed from requirements.txt make sure to install them manually

**Comment by halstead7** [comment]
como voçê consegui o ip:porta no masscan no formato certo?

Em ter., 21 de nov. de 2023 às 06:44, TechnologicalTerror <
***@***.***> escreveu:
